unpackaged/config/trial/objects/___NAMESPACE___Grade_Enrollment__c.object

```diff
@@ -67,7 +57,138 @@
+        <inlineHelpText>The reason a student exited this Grade Level.</inlineHelpText>
+        <label>Exit Reason</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Deceased</fullName>
+                    <default>false</default>
+                    <label>Deceased</label>
+                </value>
+                <value>
+                    <fullName>Expelled</fullName>
+                    <default>false</default>
+                    <label>Expelled</label>
+                </value>
+                <value>
+                    <fullName>No Show</fullName>
+                    <default>false</default>
+                    <label>No Show</label>
+                </value>
+                <value>
+                    <fullName>Promoted</fullName>
+                    <default>false</default>
+                    <label>Promoted</label>
+                </value>
+                <value>
+                    <fullName>Skipped Grade</fullName>
+                    <default>false</default>
+                    <label>Skipped Grade</label>
+                </value>
+                <value>
+                    <fullName>Transferred Out</fullName>
+                    <default>false</default>
+                    <label>Transferred Out</label>
+                </value>
+                <value>
+                    <fullName>Other</fullName>
+                    <default>false</default>
+                    <label>Other</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>%%%NAMESPACE%%%Status__c</fullName>
+        <description>The status of this Grade Level enrollment.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The status of this Grade Level enrollment.</inlineHelpText>
+        <label>Status</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Current</fullName>
+                    <default>false</default>
+                    <label>Current</label>
+                </value>
+                <value>
+                    <fullName>Former</fullName>
+                    <default>false</default>
+                    <label>Former</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
     <label>Grade Enrollment</label>
+    <deploymentStatus>Deployed</deploymentStatus>
```

Comment:
@kdy618 @jstvz @jofsky -- do we actually need this section? (lines 182 to 191). Wouldn't these fields be in the managed object info?

Reply:
just trying to get some confirmation here from @jstvz or @jofsky on whether or not some of this metadata really needs to be in the unpackaged file because we also have it in the packaged file. (The stuff on lines 182-191). Trying to clear the Dev Review column so we can get items to QA.

Reply:
@lifewithryan just pushed a revised draft can you take a look? Also can you share the quip regression docs with me?